openff/toolkit/utils/exceptions.py

```diff
@@ -101,6 +101,11 @@ class SMILESParseError(OpenFFToolkitException, ValueError):
     """The record could not be parsed into the given format"""
 
 
+# TODO: Should warnings inherit from a sort of OpenFFToolkitWarning?
```

Comment:
Yes, this is a great idea.

Reply:
Awesome - https://github.com/openforcefield/openff-toolkit/issues/1748